docs/configuring.md

```diff
@@ -38,6 +38,18 @@ counterparts:
 --8<-- ".ansible-lint"
 ```
 
+## Ignoring rules for entire files
+
+Ansible-lint will load skip rules from `tests/sanity/ignore.txt` file, if one is
+detected, the same as
+[ansible-test](https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/ignores.html).
```

Comment:
Since this feature is unrelated to `ansible-test`, I don't think it makes sense to reference it here. Not only is the behavior different, but as mentioned above, the file format is different also.